### PR TITLE
Improvements/624

### PR DESCRIPTION
**Pull request by RomanGalochkin**
I think it will be similar to this. Step by step we add functionality into this lib.

### CODE REVIEW

**Comment by OAGr** [comment]
Why did you delete ``packages/components/package.json``? We still need that.

File: packages/measurement-value/float-cdf.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const _ = require('lodash');
+
+const { MeasurementValue } = require('./measurement-value');
+
+class FloatCdf extends MeasurementValue {
+  xs(v) {
+    return v === undefined
+      ? _.get(this.input, 'xs')
+      : _.set(this.input, 'xs', v);
```
Comment (OAGr) [inline packages/measurement-value/float-cdf.js]: can you use an es6 getter/setter instead?
https://stackoverflow.com/questions/28222276/what-are-getters-and-setters-for-in-ecmascript-6-classes

File: packages/measurement-value/float-cdf.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  sizeYs() {
+    return _.size(this.ys());
+  }
+
+  validate() {
+    if (this.sizeXs() !== this.sizeYs()) {
+      throw new Error('Xs and Ys should be the same size.');
+    }
+    if (this.sizeXs() > FloatCdf.MAX_XS) {
```
Comment (OAGr) [inline packages/measurement-value/float-cdf.js]: This should be a server thing, maybe configurable here, but not forced in this lib.

File: packages/measurement-value/float-cdf.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const _ = require('lodash');
+
+const { MeasurementValue } = require('./measurement-value');
```
Comment (OAGr) [inline packages/measurement-value/float-cdf.js]: what would you think of this lib importing the ``cdf`` lib, then this class using the ``Cdf`` class underneath?

**Comment by RomanGalochkin** [comment]
> Why did you delete `packages/components/package.json`? We still need that.

Thanks, will recover.

File: packages/measurement-value/float-cdf.js (hunk 1 above)
Comment (RomanGalochkin) [reply packages/measurement-value/float-cdf.js]: I do not like setter/getters according to my own experience and knowledge they mislead clients of classes. We should write either "getXs()" or "xs()". But do not get properties directly. May this book shows it (https://www.amazon.com/Code-Complete-Practical-Handbook-Construction/dp/0735619670) may be another.

Comment (RomanGalochkin) [reply packages/measurement-value/float-cdf.js]: There are exceptions but not in this case.

Comment (OAGr) [reply packages/measurement-value/float-cdf.js]: Ok, I'm find with that.

Actually, now that I think about it, why do we even need a setter? Can't we just set these when we create a new instance. I assume they really shouldn't need to be mutated.

Comment (RomanGalochkin) [reply packages/measurement-value/float-cdf.js]: Ok, will remove setter.

File: packages/measurement-value/float-cdf.js (hunk 2 above)
Comment (RomanGalochkin) [reply packages/measurement-value/float-cdf.js]: I propose to forget about this for now, later we can fix it. Here is no much sense to write configuration providers and so on.

Comment (OAGr) [reply packages/measurement-value/float-cdf.js]: Ok, sure

File: packages/measurement-value/float-cdf.js (hunk 3 above)
Comment (RomanGalochkin) [reply packages/measurement-value/float-cdf.js]: It depends of code usage cases, but I believe that yes we can do it here.

**Comment by RomanGalochkin** [comment]
Updated.

**Comment by OAGr** [comment]
I think this is a good outline. We can add more stories to flesh it out soon/later.

**Pull request merged**